media-libs/openexr/openexr-2.5.2.ebuild

```diff
@@ -17,7 +17,7 @@ IUSE="cpu_flags_x86_avx doc examples static-libs utils test"
 RESTRICT="!test? ( test )"
 
 RDEPEND="
-	media-libs/ilmbase:=
```

Comment:
didn't openexr added in this PR by mistake? Is it any relation between this fix and `opencv`?

Reply:
This change is required to build opencv with multilib.